### PR TITLE
Add ppc64le arch

### PR DESCRIPTION
**Pull request by bells17**
- #624

### CODE REVIEW

**Review by pluser — APPROVED** [review]

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -28,10 +28,6 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version: "1.18"
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-        with:
-          platforms: linux/amd64,linux/arm64/v8
```
Comment (daichimukai) [inline .github/workflows/release.yaml:34]: @bells17 
Why did you remove only this `setup-qemu-actions` and leave remaining in other workflows? AFAIU, `setup-qemu-action` does almost the same thing as `docker run tonistiigi/binfmt`, so either one of `setup-qemu-action` or `docker run tonistiigi/binfmt` is necessary. Why not ALL `setup-qemu-action` could be removed because of `make setup`?

Comment (bells17) [reply .github/workflows/release.yaml:34]: @daichimukai Sorry, I forgot to remove it in other actions.
I removed the setup-qemu-action in other actions.

Comment (bells17) [reply .github/workflows/release.yaml:34]: @daichimukai Sorry, I forgot to remove it in other actions.
I removed the setup-qemu-action in other actions.

Comment (daichimukai) [reply .github/workflows/release.yaml:34]: @bells17 
Thank you. The diff looks good to me. Please squash the commits.

Comment (bells17) [reply .github/workflows/release.yaml:34]: @daichimukai I squashed the commits. Please review this again.

**Review by daichimukai — APPROVED** [review]
LGTM

**Review by pluser — APPROVED** [review]
